### ISSUE TITLE
Rubric for Code

### ISSUE THREAD

**Issue by nilababaei** [no-wuckas/no-wuckas-app#16]
@thomasjbradley

| | 0 points | 1 points | 2 points | 3 points |
| --- | --- | --- | --- | --- |
| **Accessibility** | No considerations | Added some alt attributes, nothing else | **Just the basics: alt attributes, roles** | Accessibility is well-considered and tested. |
| **Code quality** | Barely started. | Completely coded but full of indentation issues and validation issues, bad semantics. | Minimal indentation issues, validation issues, okay semantics. | **Pretty much perfect indentation, validation and semantics.** |
| **Images & videos** | Placeholders only. | Images used but nothing is compressed, lagging is frustrating for the user. | Images only compressed but the videos are not, lagging is still frustrating but tolerable. | **Everything compressed and is easy to manage through.** |
| **Responsive** | Not responsive at all. | Some elements are responsive such as text, but elements like images are not. | Everything is responsive but has not been tested on Chrome, Opera, Firefox, etc. | **Everything is responsive and tested.** |
| **Performance** | LAGGGGGG, get me out of here. | Lag but I’ll give the app a try. | A bit of lag but ill stay and use the app. | **No lag, bless the heavens.** |

14/15 — Overall, we did really well with the coding content. We just have a few aria details that are kinda weak but overall, it's great and working properly.

**Comment by nilababaei** [comment]
I agree with the final mark! Our project is very well done and is exactly what we wanted it to be.

Our accessibility was well done, but we had not completed all of our aria details in detail so it really is just the basics. Our code quality is amazing. We coded in a very organized manner and made sure it was simple and easy to follow if handed off to another. All our classes make sense and our CSS is all applicable. We used images in terms of icons and SVG's and they were properly placed and used only when necessary. We didn't use any video so our website will load faster. The website is responsive and it was all checked when we were tweaking up the website. The website's performance is awesome. It loads very quickly and works across different browsers.

We did an amazing job and I am very proud of the outcome. Good job team!!

**Comment by hovski** [comment]
I'm content with our final mark. Our project is well done and is exactly what we wanted it to be, minus our interactive map.

Our accessibility is complete, but we had not completed all of our aria details in detail so it really is just the basics. Our code quality is awesome. All of our coding "classes" make sense. We didn't use any videos, so our website will load extremely fast. The website is responsive and it was all checked when we were tweaking up the website. Also, our website loads quickly and effectively across different browsers.

**Comment by katz0018** [comment]
Im super happy with out final mark. I think the website is stunning, full accessibility, I could have done way better aria-details to be more in depth. The coding is fantastic, fully responsive. 

Im super proud of everyone, in the future I need to practice double and triple checking everything is done for accessibility and take more time with that.

**Comment by fkhan24** [comment]
I believe our final mark is well deserved. The web app turned out great and represents exactly what goal we are trying to achieve with its content, colours, and graphics. Our code quality is great and we were able to evenly distribute all the content for our pattern library and our main pages. Overall our group successfully completed one awesome web app and it was fairly easy with the contribution of every team member.
